### PR TITLE
OPS-52 Use bot to send status

### PR DESCRIPTION
**Pull request by LukeGuanY**
Use PR bot to send a commit status rather than making an API call.

### CODE REVIEW

File: out.txt [hunk 1]
```diff
@@ -0,0 +1,54 @@
+
```
Comment (RisePeople) [inline out.txt:1]: dont think we need this file

**Review by annc128 — APPROVED** [review]
LGTM

**Comment by RisePeople** [comment]
Luke, is this ready to replace the current prod bot?

**Comment by LukeGuanY** [comment]
> Luke, is this ready to replace the current prod bot?
No. I have one more PR which deals with the JIRA tickets is coming up.

**Pull request merged**